### ISSUE TITLE
Removed from `open-vsx/publish-extensions`

### ISSUE THREAD

**Issue by filiptronicek** [ikappas/vscode-composer#23]
Hi there,

We decided to remove this extension from the https://github.com/open-vsx/publish-extensions repository, since it has not had any activity for a while and we try to make the list of published extensions shorter.

This means that new versions of the extension will not be available on the OpenVSX registry, but feel free to enable automatic publishing yourselves, we made a pretty easy guide in our repo [[1](https://github.com/open-vsx/publish-extensions/blob/master/docs/direct_publish_setup.md)].

<details>
<summary> What is OpenVSX?</summary>

OpenVSX is a vendor neutral alternative to the MS marketplace used by most other derivatives of VS Code like [VSCodium](https://vscodium.com/), [Gitpod](gitpod.io), [OpenVSCode](https://github.com/gitpod-io/openvscode-server), [Theia](https://theia-ide.org/) based IDEs, and so on.

</details>

Feel free to close this, just wanted to let you know :)

**Comment by ikappas** [comment]
@filiptronicek I am currently working on a release

**Comment by filiptronicek** [comment]
I'm glad to hear that, @ikappas.
Please feel free to ping me if you need help with publishing, I'd love to help!

**Comment by ikappas** [comment]
@filiptronicek  Version 0.8.0 is out with several fixes, so how do we publish on open-vsix?

**Comment by filiptronicek** [comment]
@ikappas the easiest way to proceed comes down to getting an access token and claiming the namespace on OpenVSX. 

The process of creating an OpenVSX account and getting the access token is described here: https://github.com/open-vsx/publish-extensions/blob/master/docs/direct_publish_setup.md

Because we have published the extension before already, it will also be needed for you as the owner to request namespace access: https://github.com/eclipse/openvsx/wiki/Namespace-Access#how-to-claim-a-namespace. This will also make your extension marked as verified on the store.

Lastly, after we have a token we can setup a quick GitHub Action for publishing (or another way if you have a release workflow already), which is pretty customisable and can be found at https://github.com/eclipse/openvsx/wiki/Namespace-Access#how-to-claim-a-namespace.

**Comment by ikappas** [comment]
@filiptronicek I have already set up an account with open-vsx and opened an claim namespace issue.

Will the 0.8.0 version published on the VSCode Marketplace be picked up by the automated scheduled task that is run each night on open-vsx?

**Comment by filiptronicek** [comment]
@ikappas yes, since we have removed it, but only in a Pull request which is not yet merged [[1](https://github.com/open-vsx/publish-extensions/pull/565)], so the workflow ran and v0.8.0 is published there [[1](https://open-vsx.org/extension/ikappas/composer/0.8.0)].

**Comment by ikappas** [comment]
@filiptronicek Thank you for your feedback. ~~How do you revert the changes in [[1](https://github.com/open-vsx/publish-extensions/pull/565)] to exclude this repository?~~

I was prompted to request removal of the auto-publishing in order to conclude claiming the namespace:
- https://github.com/EclipseFdn/open-vsx.org/issues/942#issuecomment-1087583505
- https://github.com/open-vsx/publish-extensions/issues/574

**Comment by ikappas** [comment]
@filiptronicek Both processes are complete and I have claimed the namespace. Do you think that the pull request [[1](https://github.com/open-vsx/publish-extensions/pull/565)] should be updated to remove ikappas/* plugins from the list?

**Comment by filiptronicek** [comment]
@ikappas I have removed phpcs as well (https://github.com/open-vsx/publish-extensions/commit/415c3c5e58c7f9944cd7ffbe39e58b543be5c96d), so that both in the pull request and on `master` there is neither `composer` or `phpcs`.

If I'm correct, the only thing missing now is implementing [the GitHub workflow](https://github.com/open-vsx/publish-extensions/blob/master/docs/exampleCI.yaml) - or it can be for sure done manually if you prefer.

**Comment by ikappas** [comment]
@filiptronicek I am currently doing it manually, but I will have a look at the CI script. Thank you for your help

**Comment by ikappas** [comment]
@filiptronicek one last thing... to get rid of the warning at the top about the plugin being published by open-vsx do I need to republish the same vsix or a new version?

**Comment by filiptronicek** [comment]
@ikappas 
> @filiptronicek one last thing... to get rid of the warning at the top about the plugin being published by open-vsx do I need to republish the same vsix or a new version?

You will have to publish a new version, since this version is already published by the open-vsx account and the CLI won't let you.
